### PR TITLE
improve sdk

### PR DESCRIPTION
**Pull request by corbadoman**

### CODE REVIEW

**Review by serejja — APPROVED** [review]

File: examples/test.php [hunk 1]
```diff
@@ -1,25 +1,30 @@
...
+use Corbado\Generated\ApiException;
+use Corbado\Generated\Model\AuthTokenValidateReq;
+use Corbado\Generated\Model\AuthTokenValidateRsp;
+use Corbado\SDK;
+
 require_once __DIR__ . '/../vendor/autoload.php';
 
-$config = new \Corbado\Configuration('pro-771306541694234650', 'xxx');
-$corbado = new \Corbado\SDK($config);
+$config = new Configuration('pro-771306541694234650', 'xxx');
```
[the remark this answers is not in the document]
Comment (corbadoman) [reply examples/test.php]: i removed the file

File: examples/validation.php [hunk 2]
```diff
@@ -0,0 +1,28 @@
+<?php
+
+use Corbado\Classes\Exceptions\ServerException;
+use Corbado\Configuration;
+use Corbado\Generated\Model\ValidateEmailReq;
+use Corbado\SDK;
+
+require_once __DIR__ . '/../vendor/autoload.php';
+
+$config = new Configuration('pro-3813112311613555974', 'corbado1_VMaivnYigrGfZnlvBekASbNEk357JC');
```
[the remark this answers is not in the document]
Comment (corbadoman) [reply examples/validation.php]: i removed the file

File: src/SDK.php [hunk 3]
```diff
@@ -19,20 +27,19 @@ class SDK
...
-    private ?Widget $widget = null;
+    private ?EmailLinksInterface $emailLinks = null;
+    private ?SMSCodesInterface $smsCodes = null;
+    private ?ValidationsInterface $validations = null;
     private ?UserApi $users = null;
     private ?Session $session = null;
-    private ?AuthTokensApi $authTokens = null;
+    private ?AuthTokensInterface $authTokens = null;
+
+    const VERSION = '1.0.0';
```
[the remark this answers is not in the document]
Comment (corbadoman) [reply src/SDK.php]: we are already 2.x so for this update i need to go to 3.x because the interface changed

**Review by corbadovych — APPROVED** [review]
looks good :)

**Review by incorbador — APPROVED** [review]
LGTM

**Pull request merged**